src/app/navigation/application-launcher.component.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Component, ElementRef, EventEmitter,
+    Input, Output, Renderer2, TemplateRef,
+    ViewEncapsulation
+} from '@angular/core';
+
+import { NavigationItemConfig } from './navigation-item-config';
+
+
+
+/**
+ * Application Launcher component
+ * @param {boolean=} isDisabled Disable the application launcher button, default: false
+ * @param {boolean=} isList Display items as a list instead of a grid, default: false
+ * @param {boolean=} hiddenIcons Flag to not show icons on the launcher, default: false
```

Comment:
The inputs are documented below (the docs will pick that up), so it's not necessary to add that here.

Reply:
@dlabrecq : sure, I will remove it.